lib/src/text-input/TextInput.test.js

```diff
@@ -524,7 +518,7 @@ describe("TextInput component synchronous autosuggest tests", () => {
       <DxcTextInput label="Autocomplete Countries" value="Andor" suggestions={countries} onChange={onChange} />
     );
     const input = getByRole("combobox");
-    fireEvent.focus(input);
+    await userEvent.click(getByText("Autocomplete Countries"));
```

Comment:
There's no need for an `await` here, it works fine without it. In that case, we can also remove the async.